src/pages/miyeon/Main/components/mainFeed/MainFeed.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import Comment from '../feedComment/Comment';
+
+class Mainfeed extends Component {
+  constructor() {
+    super();
+    this.state = {
+      heartBtn: false,
+      likeNumber: 2,
+    };
+  }
+
+  heartBtnActive = () => {
+    this.setState({
+      heartBtn: !this.state.heartBtn,
+    });
+  }; // 좋아요 기능 - 하트 클릭시 진한 컬러 하트로 변경
+
+  likeNumberPlus = () => {
+    const { heartBtn, likeNumber } = this.state;
+    heartBtn === true
```

Comment:
```suggestion
    heartBtn
```
=== true 랑 비교 안하셔도 boolean 값 반환